Source/SwiftLintFramework/Reporters/HTMLReporter.swift

```diff
@@ -25,16 +25,25 @@ public struct HTMLReporter: Reporter {
         return "Reports violations as HTML"
     }
 
-    // swiftlint:disable:next function_body_length
     public static func generateReport(_ violations: [StyleViolation]) -> String {
+        let dateString = formatter.string(from: Date())
+        return generateReport(
+                violations,
+                swiftlintVersion: swiftlintVersion,
+                dateString: dateString
+        )
+    }
+
+    // swiftlint:disable function_body_length
+    // swiftlint:disable line_length
```

Comment:
yeah, could you please file a new ticket for this? I don't think my quick implementation from last week is 100% correct.